template/project.json

```diff
@@ -1,20 +1,29 @@
 {
-    "version": "1.0.0-*",
-    "compilationOptions": {
-        "emitEntryPoint": true
-    },
-
-    "compilerName": "fsc",
-    "compileFiles": [
-        "Program.fs"
-    ],
+  "version": "1.0.0-*",
+  "compilationOptions": {
+    "emitEntryPoint": true
+  },
+  "compilerName": "fsc",
+  "compileFiles": [
+    "Program.fs"
+  ],
+  "dependencies": {
+    "Microsoft.FSharp.Core.netcore": "1.0.0-alpha-160329",
+    "Microsoft.NETCore.App": {
+      "version": "1.0.0-rc2-23931"
+    }
+  },
 
-    "dependencies": {
-        "Microsoft.FSharp.Core.netcore": "1.0.0-alpha-160318",
-        "NETStandard.Library": "1.5.0-rc2-23911"
-    },
+  "tools": {
+       "dotnet-compile-fsc": "1.0.0-*"
```

Comment:
You should add the imports here similar to the change in https://github.com/dotnet/cli/pull/2214